include/glow/Backends/BackendUtils.h

```diff
@@ -34,6 +35,12 @@ size_t
 getValueOffset(Value *v,
                const std::unordered_map<std::string, runtime::RuntimeSymbolInfo>
                    &symbolTable);
+/// Computes offsets and total allocation for Constants, Placeholders, and
+/// Activations to build runtime symbol table. Returns RuntimeBundle.
+runtime::RuntimeBundle
+generateRuntimeBundle(const IRFunction *F, MemoryAllocator *constantAllocator,
+                      MemoryAllocator *placeholderAllocator,
+                      MemoryAllocator *activationsAllocator);
```

Comment:
Does the memory allocator or bundle have a notion of out-of-memory?
I.e., where would we fail if we try to allocate a bundle that won't fit on the HW?

Reply:
Currently we assume it will fit at compile time. We would fail at runtime allocation if there is insufficient memory.